script.js

```diff
@@ -37,35 +41,52 @@ $(document).ready(function(){
 
   $(".target").mouseup(function () {
     if (click == true){
-      release();
+      release($(this));
     }
   });
 
   $(".target").mouseout(function () {
     if (click == true){
-      release();
+      release($(this));
     }
   });
 
-  var release = function(){
-    deActive($('.light'));
-    attempts++;
+  var release = function(curClick){
     click = false;
-    console.log('attempts',attempts,'turn',turn);
-    if(attempts == turn){
+    deActive($('.light'));
+
+    if(!curClick.hasClass(lightSeq[attempt-1].toString())){
```

Comment:
I would consider encapsulating this game logic in its own object (as noted above) - as the code is currently written, is it difficult to understand the intent.  If we move this into a `Game` object that is responsible for maintaining and mutating state, we could create a more meaningful, intention-revealing function name to facilitate understanding of the code (which helps with maintenance and code changes).